elasticsearch_async/connection.py

```diff
@@ -6,12 +6,14 @@
 from elasticsearch.exceptions import ConnectionError, ConnectionTimeout, SSLError
 from elasticsearch.connection import Connection
 from elasticsearch.compat import urlencode
+from elasticsearch.connection.http_urllib3 import create_ssl_context
```

Comment:
I think this was only added in certain version, we should update `setup.py` to reflect that...

Reply:
@HonzaKral thanks will update. 

need to get the tests running locally. Since it appears we don't have travis hooke dup here